### DISCUSSION TITLE
Idea / proposal - Vote Points

### DISCUSSION THREAD

**Discussion by Wrath-Man** [chromiecraft/chromiecraft#4121] [Suggestions]
Hello,

I have an idea to increase the population of people on our server. 

To do this, I propose the introduction of Vote Points for casting votes. 

Adding in My account in the panel the Vote Points option and a list of pages with voting addresses. Introduction of validation of the correctness of voting and time limits. 

Determination of scores and prizes for casting votes. Proposal of cosmetic gifts e.g. mounts. 

https://www.chromiecraft.com/en/vote-us/# 

P.S The first page probably does not work to cast a vote on chrome, the lack on the page is not in the search engine. 

https://topg.org/wow-private-servers/

I realize that the implementation of such a system requires work and the administration has a lot of other activities but I think that it would increase the popularity of this server and new players would be very happy to join the Chrome project where you can not buy in the store gold and items affecting the game (without p2w) which gives a huge advantage over other servers and development potential. 

Of course, I am in favor of grants supported by the maintenance and development of this project, which is why I propose that for the addition you can get ONLY cosmetic items, e.g. mounts.

**Comment by balleny** [comment]
Unfortunately, those systems have been regularly exploited year over year.
Also, highest visibility isnt always what you want (see Turtle WoW in recent time)

**Comment by gettheboom** [comment]
I vote whenever I remember about it... But making it rewarding will motivate people to vote more.
